### DISCUSSION TITLE
ansible -mysql ( Disable MySQL 8 Version )

### DISCUSSION THREAD

**Discussion by chowdary709** [learndevopsonline/learndevopsonline#1623] [Q&A]
hi raghu garu 
- name: Disable MySQL 8 Version
  ansible.builtin.shell: dnf module disable mysql -y

this command multiple times run  this error message will showing
==============================================
TASK [02.mysql : Disable MySQL 8 Version] **********************************************************************************************************
fatal: [mysql.roboshop.internal]: FAILED! => {"changed": true, "cmd": "dnf module disable mysql -y", "delta": "0:00:13.730512", "end": "2023-11-21 06:17:04.023391", "msg": "non-zero return code", "rc": -6, "start": "2023-11-21 06:16:50.292879", "stderr": "Out of memory allocating 301989888 bytes!", "stderr_lines": ["Out of memory allocating 301989888 bytes!"], "stdout": "", "stdout_lines": []}
===============================================
- name: Disable MySQL 8 Version
  ansible.builtin.service:
    name: mysqld
    state: stopped
    enabled: no
use this module cannot shown error

**Comment by sreedharm07** [comment]
@chowdary709 
ping your ip and please attach error screenshot
